### ISSUE TITLE
Update models

### ISSUE THREAD

**Issue by briatte** [briatte/epsa2018#3]
- [ ] Update `frm` models to (heteroskedasticity-corrected) panels
- [ ] Also have a look at naive logits
- [ ] Also have a look at beta regressions
